pkg/containertools/dockerfilegenerator.go

```diff
@@ -48,7 +48,7 @@ func (g *IndexDockerfileGenerator) GenerateIndexDockerfile(binarySourceImage, da
 	dockerfile += fmt.Sprintf("LABEL %s=%s\n", DbLocationLabel, DefaultDbLocation)
 
 	// Content
-	dockerfile += fmt.Sprintf("ADD %s/index.db %s\n", databaseFolder, DefaultDbLocation)
```

Comment:
This was still just adding a single file, right? I'm not sure how that could have been including the extra context.

Reply:
It was including it in the docker build context, not the final image.

Reply:
I see now, thanks.